Header header

float32 ave_deviation_z
float32 ave_horizontal_reprojecterr
float32 ave_vertical_reprojecterr